br/backup-and-restore-storages.md

```diff
@@ -149,7 +149,7 @@ When storing backup data in a cloud storage system, you need to configure authen
 Before backup, configure the following privileges to access the backup directory on S3.
 
 - Minimum privileges for TiKV and Backup & Restore (BR) to access the backup directories during backup: `s3:ListBucket`, `s3:PutObject`, and `s3:AbortMultipartUpload`
-- Minimum privileges for TiKV and BR to access the backup directories during restore: `s3:ListBucket` and `s3:GetObject`
+- Minimum privileges for TiKV and BR to access the backup directories during restore: `s3:ListBucket`, `s3:GetObject` and `s3:PutObject`(BR will write the checkpoint information to the `/checkpoints` subdirectory under the backup directory; When BR restores the log backup files, it will write the rewrite id map to the `/pitr_id_maps` subdirectory under the backup directory.)
```

Comment:
```suggestion
- Minimum privileges for TiKV and BR to access the backup directories during restore: `s3:ListBucket`, `s3:GetObject` and `s3:PutObject`(BR will write the checkpoint information to the `./checkpoints` subdirectory under the backup directory; When BR restores the log backup files, it will write the rewrite id map to the `./pitr_id_maps` subdirectory under the backup directory.)
```

Reply:
Or we remove the / at all. Please also update the Chinese version.

Reply:
done